### ISSUE TITLE
Docker Dev Image not working on fly.io anymore

### ISSUE THREAD

**Issue by colin** [jointakahe/takahe#463]
It seems like the most recent Docker dev images (https://hub.docker.com/r/jointakahe/takahe-dev/tags) can't be deployed to fly.io anymore (and it might be similar for AWS Lambda and Google Cloud Run). 

I am getting an `Could not find image "docker.io/jointakahe/takahe-dev:sha-ee3b51c"` error from fly.io which seems to be based on the fact that `docker manifest inspect jointakahe/takahe-dev:sha-ee3b51c` returns a `no such manifest` error. Given that everything works for the 0.7.0 Docker image, I tried to go down the rabbit hole that Github Actions, Docker image builds and the underlying build systems unfortunately seem to be. My (somewhat vague) understanding is that Github very recently updated their Ubuntu Actions runner (https://github.com/actions/runner-images/pull/6941) which included an update to docker-buildx (0.9.1 to 0.10.0) which outputs image manifest in a new/different way that `docker manifest` can't read at the moment (https://github.com/docker/buildx/issues/1509) which results in the various container hosting services not being able to run the images (https://github.com/docker/buildx/issues/1533). 

A possible solution might be reverting to the previous docker-buildx version like this (https://github.com/docker/build-push-action/issues/755#issuecomment-1378680855):
```
  - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2
        with:
          version: v0.9.1
```

I'm not entirely sure what other implications that might have and there are also other ways this could be fixed (including fly.io and other services changing/updating how they are pulling image info from Docker Hub), but maybe it's an easy fix to keep the dev images easily deployable on these kinds of services?

**Comment by andrewgodwin** [comment]
Alright, I've tried that fix and set a new Docker build in motion - it should finish in about 20 mins. Let me know if that works?

**Comment by colin** [comment]
Seems to be fixed with 349e1b8801fcfbb273c704fcc1a5c7771d2d915c.
`docker manifest` output looks like before and fly.io can find/deploy the image again.
